### PR TITLE
Rename behaviour methods

### PR DESCRIPTION
**Pull request by gsferreira**

### CODE REVIEW

File: docs/pages/omnia3/Modeler/omnia3_modeler_bestpractices.md [hunk 1]
```diff
@@ -61,3 +61,14 @@ Decimal attributes that are calculated on C# behaviours should be rounded to mat
 ```
     this.Average = Decimal.Round(this.Amount / this.Quantity, 2);
 ```
+
+### C# - Use async/await when is possible
+
+When working with c# Tasks using ".GetAwaiter().GetResult()", ".Result" or ".Wait()" to get the result cam cause [deadlocks or thread pool starvation](https://gsferreira.com/archive/2020/08/avoid-getawaiter-getresult-at-all-cost/). So, it's recommended to use async/await all way down.
```
Comment (zebilha) [inline docs/pages/omnia3/Modeler/omnia3_modeler_bestpractices.md:67]: to get the result **can** cause

Comment (zebilha) [inline docs/pages/omnia3/Modeler/omnia3_modeler_bestpractices.md:67]: to get the result **cam** cause

**Review by zebilha — CHANGES_REQUESTED** [review]

**Review by zebilha — APPROVED** [review]

**Review by daniel8lopes — APPROVED** [review]

File: docs/pages/omnia3/Modeler/omnia3_modeler_bestpractices.md [hunk 2]
```diff
@@ -61,3 +61,14 @@ Decimal attributes that are calculated on C# behaviours should be rounded to mat
...
+
+In cases like Initialize behaviours or Action/Change behaviours, you can't use async/await. On those cases, you should extract async methods to Code Dependencies and use "GetAwaiter().GetResult()" just once to wait for that Code Dependency.
```
Comment (daniel8lopes) [inline docs/pages/omnia3/Modeler/omnia3_modeler_bestpractices.md:69]: In cases like Initialize behaviours or Action/Change behaviours, you can't use async/await. In these cases, you should extract async methods to Code Dependencies and use "GetAwaiter().GetResult()" just once to wait for that Code Dependency.

**Review by daniel8lopes — COMMENTED** [review]
Commented correction

**Review by daniel8lopes — APPROVED** [review]

**Review by daniel8lopes — APPROVED** [review]

**Pull request merged**